staging/src/k8s.io/api/admissionregistration/v1alpha1/types.go

```diff
@@ -240,6 +272,15 @@ type ValidatingAdmissionPolicyBindingSpec struct {
 	// Note that this is differs from ValidatingAdmissionPolicy matchConstraints, where resourceRules are required.
 	// +optional
 	MatchResources *MatchResources `json:"matchResources,omitempty" protobuf:"bytes,3,rep,name=matchResources"`
+
+	// ValidationActions declares how Validations of the referenced ValidatingAdmissionPolicy are enforced.
+	// Validations that evaluate to false are enforced according to ValidationActions.
+	// Failures defined by the ValidatingAdmissionPolicy's FailurePolicy are enforced according
+	// to ValidationActions only if the FailurePolicy is set to Fail, otherwise the failures are ignored. This
+	// includes compilation errors, runtime errors and misconfigurations of the policy.
+	// Required.
+	// +listType=atomic
+	ValidationActions []ValidationAction `json:"validationActions,omitempty" protobuf:"bytes,4,rep,name=validationActions"`
```

Comment:
if we want this enum to be able to have additional values in the future, document that possibility and how clients should handle unknown enum values (xref https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api_changes.md#new-enum-value-in-existing-field)